SU2_CFD/src/output_structure.cpp

```diff
@@ -3859,21 +3861,35 @@ void COutput::SetRestart(CConfig *config, CGeometry *geometry, CSolver **solver,
     }
     restart_file << "\n";
   }
-  
-  /*--- Write the general header and flow conditions ----*/
-  
-  restart_file <<"AOA= " << config->GetAoA() - config->GetAoA_Offset() << endl;
-  restart_file <<"SIDESLIP_ANGLE= " << config->GetAoS() - config->GetAoS_Offset() << endl;
-  restart_file <<"INITIAL_BCTHRUST= " << config->GetInitial_BCThrust() << endl;
-  restart_file <<"DCD_DCL_VALUE= " << config->GetdCD_dCL() << endl;
-  if (adjoint) restart_file << "SENS_AOA=" << solver[ADJFLOW_SOL]->GetTotal_Sens_AoA() * PI_NUMBER / 180.0 << endl;
-  if (dual_time)
-    restart_file <<"EXT_ITER= " << config->GetExtIter() + 1 << endl;
-  else
-    restart_file <<"EXT_ITER= " << config->GetExtIter() + config->GetExtIter_OffSet() + 1 << endl;
-  
+
+  /*--- Close the data portion of the restart file. ---*/
+
   restart_file.close();
-  
+
+  /*--- Also create the filename for the restart metadata ---*/
+
+  if (wrt_metadata) {
```

Comment:
The metafile is only necessary if you want to include the ExtIter, AoA, etc. 

However, I have decided to put this information back into the restart files with the next PR, so this will disappear. Everything with the restarts will work as always (file names, extensions, metadata, etc.) with the change to binary being transparent to the user.

I'll leave this as is for now, since the next PR changes it again.